### PR TITLE
Implement /user/show viewer and /user/:id route.

### PR DESCRIPTION
**Pull request by webdevjeffus**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  rounds.each do |round|
+    # Deck is already in the hash
+    if decks_played.has_key?(round.id)
+      decks_played[round.id] << round
+    # Deck is not in the hash
+    else
+      decks_played[round.id] = []
+      decks_played[round.id] << round
+    end
+
```
Comment (dijonkitchen) [inline app/controllers/users_controller.rb:19]: Missing end for each loop

File: app/views/users/show.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<div class="container">
+  <h2>UserName</h2>
+
+  <!-- Iterate over each kind of deck -->
+  <% decks_played.each do |round_id, rounds_played| %>
+    <h3>Deck Title</h3>
+    <table>
```
Comment (dijonkitchen) [inline app/views/users/show.erb:7]: Tables are 1999.

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (elizaway) [inline app/controllers/users_controller.rb:19]: Looks good.

Comment (webdevjeffus) [inline app/controllers/users_controller.rb:19]: end added at line 18.

File: app/views/users/show.erb (hunk 2 above)
Comment (webdevjeffus) [inline app/views/users/show.erb:7]: Tonight I'm gonna party like same.

**Pull request merged**